desktop/core/src/desktop/lib/conf_test.py

```diff
@@ -179,25 +180,24 @@ def test_set_for_testing(self):
       close()
     assert "baz_default" == self.conf.SOME_SECTION.BAZ.get()
 
-
   def test_coerce_bool(self):
-    assert False == coerce_bool(False)
-    assert False == coerce_bool("FaLsE")
-    assert False == coerce_bool("no")
-    assert False == coerce_bool("0")
-    assert True == coerce_bool("TrUe")
-    assert True == coerce_bool("YES")
-    assert True == coerce_bool("1")
-    assert True == coerce_bool(True)
+    assert not coerce_bool(False)
+    assert not coerce_bool("FaLsE")
+    assert not coerce_bool("no")
+    assert not coerce_bool("0")
+    assert coerce_bool("TrUe")
+    assert coerce_bool("YES")
+    assert coerce_bool("1")
+    assert coerce_bool(True)
     with pytest.raises(Exception):
       coerce_bool(tuple("foo"))
 
   def test_print_help(self):
     out = string_io()
     self.conf.print_help(out=out, skip_header=True)
     out = out.getvalue().strip()
-    assert not "dontseeme" in out
-    assert re.sub("^    (?m)", "", """
+    assert "dontseeme" not in out
+    assert re.sub("(?m)^    ", "", """
```

Comment:
Is there an official guideline doc from python.org? Please share the source.

Reply:
Shared in the description already.
1. https://bugs.python.org/issue47066
2. https://github.com/python/cpython/issues/91222
3. https://github.com/python/cpython/pull/31994